packages/tasks/src/local-apps.ts

```diff
@@ -180,6 +183,33 @@ const snippetVllm = (model: ModelData): LocalAppSnippet[] => {
 		},
 	];
 };
+const snippetTgi = (model: ModelData): LocalAppSnippet[] => {
+	const runCommand = [
+		"# Call the server using curl:",
+		`curl -X POST "http://localhost:8000/v1/chat/completions" \\`,
+		`	-H "Content-Type: application/json" \\`,
+		`	--data '{`,
+		`		"model": "${model.id}",`,
+		`		"messages": [`,
+		`			{"role": "user", "content": "What is the capital of France?"}`,
+		`		]`,
+		`	}'`,
+	];
+	return [
+		{
+			title: "Use Docker images",
+			setup: [
+				"# Deploy with docker on Linux:",
```

Comment:
will there be one snippet section per OS?

Reply:
I was thinking of just having the linux one here (same as vllm)

Reply:
ah yes but that's just the guest OS, not the host, right?

Reply:
Oui! but, we use `http://localhost:8000/v1/chat/completions` in the usage snippet so it'll make it a bit complicated to have a different host.